example/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
+    "android": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res && react-native run-android",
```

Comment:
Any reason to change this?

Reply:
@akshaybhalotia we are getting issue with caching where all cached store in bundle , if you want to run app in android this file is needed and need to clean before build the app in android device so that we added this command .

Reply:
Got it